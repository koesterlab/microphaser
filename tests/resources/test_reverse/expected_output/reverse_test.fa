>a324f17d17b218dR
cGAGCCACACCCATCACTCAACATGTC
>c258647e023393eR
TTCcGAGCCACACCCATCACTCAACAT
>172c41a2e6e2cb7R
TTCTTCcGAGCCACACCCATCACTCAA
>b5f424dc651bf48R
CTTTTCTTCcGAGCCACACCCATCACT
>965e45175515136R
GATCTTTTCTTCcGAGCCACACCCATC
>08add1803a50f62R
TGAGATCTTTTCTTCcGAGCCACACCC
>223bd8fb09288ccR
GACTGAGATCTTTTCTTCcGAGCCACA
>732c0f3ff28e5c0R
AGGGACTGAGATCTTTTCTTCcGAGCC
>58c2ffd43739c73R
GGAAGGGACTGAGATCTTTTCTTCcGA
>742163ac1cebc12R
CcGCTGCCTGCCCACCACACGGCCCTC
>fd764a5345f7473R
CATCcGCTGCCTGCCCACCACACGGCC
>a55d1f2587eae5cR
GTGCATCcGCTGCCTGCCCACCACACG
>ae580a513507433R
CTTGTGCATCcGCTGCCTGCCCACCAC
>946de2980abc10fR
GGCCTTGTGCATCcGCTGCCTGCCCAC
>4d2f7903355cae3R
CAAGGCCTTGTGCATCcGCTGCCTGCC
>8a68b9eec214f8bR
GTCCAAGGCCTTGTGCATCcGCTGCCT
>56239f5beba4a30R
CCTGTCCAAGGCCTTGTGCATCcGCTG
>c8cd1f33758a3fcR
CACCCTGTCCAAGGCCTTGTGCATCcG
>0bf5651d15c846cR
acAgggactggggccgggaccgggaca
>683eeee967ad8d0R
gggacAgggactggggccgggaccggg
>7023ad3880e2d57R
accgggacAgggactggggccgggacc
>6f5b9cce754f8f2R
gggaccgggacAgggactggggccggg
>5302a9716f887d3R
gccgggaccgggacAgggactggggcc
>c78a8dc5274f849R
ggggccgggaccgggacAgggactggg
>abb5a5249057784R
actggggccgggaccgggacAgggact
>8aaf4c7d467b4f4R
gggactggggccgggaccgggacAggg
>18fa98f4b970791R
accgggactggggccgggaccgggacA
>f7a0e246701c5ddR
acAgggactggggccgggaccgggacA
