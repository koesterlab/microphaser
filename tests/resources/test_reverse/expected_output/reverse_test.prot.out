>acf38ffbf73913dR
GCTAAGGGAGGCCAAAGGTGAGCTCAT
>2bbea03c534d232R
CTTGCTAAGGGAGGCCAAAGGTGAGCT
>52ef87c34245fe6R
GGTCTTGCTAAGGGAGGCCAAAGGTGA
>ca2ed43d071f450R
TCGGGTCTTGCTAAGGGAGGCCAAAGG
>100372267bef937R
TTTTCGGGTCTTGCTAAGGGAGGCCAA
>981090c05739fddR
CACTTTTCGGGTCTTGCTAAGGGAGGC
>ddb9d3948eabe24R
GGGCACTTTTCGGGTCTTGCTAAGGGA
>a7e39cfb0aacc0aR
CAGGGGCACTTTTCGGGTCTTGCTAAG
>25fbfa138bb9bc8R
GGGCAGGGGCACTTTTCGGGTCTTGCT
>10526b6623da683R
CGAGGGCAGGGGCACTTTTCGGGTCTT
>091f04f41e45bb2R
CTCCGAGGGCAGGGGCACTTTTCGGGT
>8888ddb3736ed83R
AGGCTCCGAGGGCAGGGGCACTTTTCG
>f02412d37d8ea10R
CATAGGCTCCGAGGGCAGGGGCACTTT
>224c88e8226e8e2R
ATTCATAGGCTCCGAGGGCAGGGGCAC
>1b847786a95b9feR
AGGATTCATAGGCTCCGAGGGCAGGGG
>ecead6b5b7f5b4aR
ATCTCCATAGATGCGGATTCCTCGCCT
>cb7f2559234bc5aR
CCCATCACTCAACATGTCCACCTCATC
>5ac3506620f6adfR
ATCACTCAACATGTCCACCTCATCTTC
>71ed5d1cdf0a590R
ACTCAACATGTCCACCTCATCTTCATC
>e8c5e26e36ef11aR
CAACATGTCCACCTCATCTTCATCTCC
>b829bb7e37de6c8R
CATGTCCACCTCATCTTCATCTCCATA
>f5a948e7d8b24aeR
GTCCACCTCATCTTCATCTCCATAGAT
>885542933efc713R
CACCTCATCTTCATCTCCATAGATGCG
>dbe9236207a7341R
CTCATCTTCATCTCCATAGATGCGGAT
>6e8da82153f5051R
ATCTTCATCTCCATAGATGCGGATTCC
>cd2c7cd2f0936e7R
ACACCCATCACTCAACATGTCCACCTC
>d5e6c37e77a39b4R
GCCACACCCATCACTCAACATGTCCAC
>5ddecedfd81079aR
GCAGGAAGGGACTGAGATCTTTTCTTC
>7492308112b646eR
ATAGCAGGAAGGGACTGAGATCTTTTC
>e4e3b1720da5141R
GCCATAGCAGGAAGGGACTGAGATCTT
>320cc00de6c5ecdR
GCCGCCATAGCAGGAAGGGACTGAGAT
>68faef1f79dee9eR
TATGCCGCCATAGCAGGAAGGGACTGA
>dcb1e0bb54a5614R
ACCTATGCCGCCATAGCAGGAAGGGAC
>96080a749b4340dR
GGCACCTATGCCGCCATAGCAGGAAGG
>2d8df57b6c87c17R
AGGGGCACCTATGCCGCCATAGCAGGA
>3168678c2a3a9cdR
CACAGGGGCACCTATGCCGCCATAGCA
>839409274d84e9bR
ACTCACAGGGGCACCTATGCCGCCATA
>150911f20600496R
CCGACTCACAGGGGCACCTATGCCGCC
>cd84353ad68be53R
TTGCCGACTCACAGGGGCACCTATGCC
>22b09d966683f7bR
GTCCACCTCAGGGGGCGCCAATGAGTC
>5d9b4da1613ec45R
CACCTCAGGGGGCGCCAATGAGTCCCC
>b9f7d30a3b835ebR
CTCAGGGGGCGCCAATGAGTCCCCTTG
>b4199677830e470R
AGGGGGCGCCAATGAGTCCCCTTGCCG
>363bbfce6999bdbR
GGGCGCCAATGAGTCCCCTTGCCGACT
>46d149381227d82R
CGCCAATGAGTCCCCTTGCCGACTCAC
>f57b55cbcc9e888R
CAATGAGTCCCCTTGCCGACTCACAGG
>e586b0d385d31b7R
TGAGTCCCCTTGCCGACTCACAGGGGC
>c82f08ab80df33eR
GTCCCCTTGCCGACTCACAGGGGCACC
>3067b3f0b70baadR
AAAGTCCACCTCAGGGGGCGCCAATGA
>50177ec8bf2b33cR
GTCAAAGTCCACCTCAGGGGGCGCCAA
>83c8ffc9aa3061bR
CCTGTCAAAGTCCACCTCAGGGGGCGC
>19886c380c039c3R
CAGCCTGTCAAAGTCCACCTCAGGGGG
>fa5a46258d856e0R
CAGCAGCCTGTCAAAGTCCACCTCAGG
>11ceef453253bf2R
GGCCAGCAGCCTGTCAAAGTCCACCTC
>425aaf3d19b8d62R
GCTGGCCAGCAGCCTGTCAAAGTCCAC
>171db2101a05903R
CAGGCTGGCCAGCAGCCTGTCAAAGTC
>2b5534704551c6eR
CTGCAGGCTGGCCAGCAGCCTGTCAAA
>4ee6bb391f83df4R
ATCCTGCAGGCTGGCCAGCAGCCTGTC
>cadd9049372e419R
AAGATCCTGCAGGCTGGCCAGCAGCCT
>ac908757133a575R
ACTAAGATCCTGCAGGCTGGCCAGCAG
>65e31db47168e5dR
CTCACTAAGATCCTGCAGGCTGGCCAG
>a68ee5baeea332aR
TGTCACTTGGGTGTCACCCTCTACCAC
>bf6b4a9caf9bd4cR
TGGTGTCACTTGGGTGTCACCCTCTAC
>3fdb5a80283b6edR
CACTGGTGTCACTTGGGTGTCACCCTC
>d293649bd057b74R
GGGCACTGGTGTCACTTGGGTGTCACC
>111c661aeeac324R
GCCGGGCACTGGTGTCACTTGGGTGTC
>5363a238b4ba0e7R
CCCGCCGGGCACTGGTGTCACTTGGGT
>b7865a5ff8591b2R
TGCCCCGCCGGGCACTGGTGTCACTTG
>e3d3a59e852c4f9R
CCGTGCCCCGCCGGGCACTGGTGTCAC
>23c3dcc507661a4R
CAGCCGTGCCCCGCCGGGCACTGGTGT
>2fe1528e48af173R
ACGCAGCCGTGCCCCGCCGGGCACTGG
>3e98a8c23d046f2R
GGTACGCAGCCGTGCCCCGCCGGGCAC
>f69ea44c61f56b5R
GAGGGTACGCAGCCGTGCCCCGCCGGG
>68373a1dcd9e7c1R
CTCGAGGGTACGCAGCCGTGCCCCGCC
>2a54c091e24e281R
GGGCTCGAGGGTACGCAGCCGTGCCCC
>3cf4a4723560e6dR
GATGGGCTCGAGGGTACGCAGCCGTGC
>7741cdca4d7fdbcR
CGGGATGGGCTCGAGGGTACGCAGCCG
>1ee4dfec2ad6818R
CAGCGGGATGGGCTCGAGGGTACGCAG
>581ad86a4e9be25R
CTTCAGCGGGATGGGCTCGAGGGTACG
>4fc7e07448eb0b2R
GAGCTTCAGCGGGATGGGCTCGAGGGT
>4739e09fefd76d4R
GTAGAGCTTCAGCGGGATGGGCTCGAG
>01c93d1e8b5223fR
CCGGTAGAGCTTCAGCGGGATGGGCTC
>b80f46c97da7743R
ATTCCGGTAGAGCTTCAGCGGGATGGG
>7ec07fcb570c74eR
GCCATTCCGGTAGAGCTTCAGCGGGAT
>d72f2f7fdf5e181R
GATGCCATTCCGGTAGAGCTTCAGCGG
>2ff7efbb2be4a28R
CATGATGCCATTCCGGTAGAGCTTCAG
>9110336744e0f9bR
CATCATGATGCCATTCCGGTAGAGCTT
>598e1755be9957dR
GAACATCATGATGCCATTCCGGTAGAG
>34d39e5383b8af8R
GTCGAACATCATGATGCCATTCCGGTA
>c683d357b5e08a3R
CCCGTCGAACATCATGATGCCATTCCG
>8b29870144514c1R
GGGCCCGTCGAACATCATGATGCCATT
>bb520ce1fac8e9dR
GAAGGGCCCGTCGAACATCATGATGCC
>a84f04007726c62R
CTGGAAGGGCCCGTCGAACATCATGAT
>2252bce22480a57R
GGGCTGGAAGGGCCCGTCGAACATCAT
>8e721f10ec28218R
GAAGGGCTGGAAGGGCCCGTCGAACAT
>d846214ce1d7f91R
GTAGAAGGGCTGGAAGGGCCCGTCGAA
>90c3cfaafa2eb1dR
ATCGTAGAAGGGCTGGAAGGGCCCGTC
>246991e7e96b936R
GGGATCGTAGAAGGGCTGGAAGGGCCC
>34db868f42f3b28R
GGAGGGATCGTAGAAGGGCTGGAAGGG
>5104ede61682ae5R
TGTGGAGGGATCGTAGAAGGGCTGGAA
>5a84a4472713c7bR
CTGTGTGGAGGGATCGTAGAAGGGCTG
>0beeb5d36d817c9R
GCCATCCAATATGTCTCGGAGGCAGCG
>c00f968171d10f7R
ATCCAATATGTCTCGGAGGCAGCGCTG
>78e0d1cb5b4bf43R
CAATATGTCTCGGAGGCAGCGCTGTGT
>041d41a82da0f90R
TATGTCTCGGAGGCAGCGCTGTGTGGA
>cb330b59480b691R
GTCTCGGAGGCAGCGCTGTGTGGAGGG
>504fc225a18b212R
TCGGAGGCAGCGCTGTGTGGAGGGATC
>23a6db5d7e2e14aR
GAGGCAGCGCTGTGTGGAGGGATCGTA
>b7c21bad4540656R
GCAGCGCTGTGTGGAGGGATCGTAGAA
>408c97ef6d515e7R
GAAGCCATCCAATATGTCTCGGAGGCA
>65b37d9c37f0abfR
AAAGAAGCCATCCAATATGTCTCGGAG
>b1b71b357d2bce2R
GGGAAAGAAGCCATCCAATATGTCTCG
>d5c3aacf7c52f9fR
TGAGGGAAAGAAGCCATCCAATATGTC
>c97c2bd7c4079c4R
CTCTGAGGGAAAGAAGCCATCCAATAT
>e2eac74204ae04eR
GAGCTCTGAGGGAAAGAAGCCATCCAA
>f0fca304832c545R
CTGGAGCTCTGAGGGAAAGAAGCCATC
>6d76e2ae5e95de8R
TCGCTGGAGCTCTGAGGGAAAGAAGCC
>b7f347a64fab8caR
CAGTCGCTGGAGCTCTGAGGGAAAGAA
>0a442d1ab70452cR
GTACAGTCGCTGGAGCTCTGAGGGAAA
>2bab2debe7e1972R
GGGGTACAGTCGCTGGAGCTCTGAGGG
>875fa2371b1c9f8R
ATTGGGGTACAGTCGCTGGAGCTCTGA
>54129d08e294156R
CCCATTGGGGTACAGTCGCTGGAGCTC
>384f4767a7a6695R
GACCCCATTGGGGTACAGTCGCTGGAG
>05c703ed61aa49cR
GGGGACCCCATTGGGGTACAGTCGCTG
>640610256e8f29eR
AAAGGGGACCCCATTGGGGTACAGTCG
>6327811f008cb52R
CTTAAAGGGGACCCCATTGGGGTACAG
>0c61d08ef81368dR
GTAGACCTGATTGCGCAAGTCACTCAC
>d6697be958b03a1R
GACCTGATTGCGCAAGTCACTCACCTT
>193191255642695R
CTGATTGCGCAAGTCACTCACCTTAAA
>b3cd6858910d5a7R
ATTGCGCAAGTCACTCACCTTAAAGGG
>a3d829e3ee69151R
GCGCAAGTCACTCACCTTAAAGGGGAC
>8af29f9e3400efbR
CAAGTCACTCACCTTAAAGGGGACCCC
>fee2580c7c7a245R
GTCACTCACCTTAAAGGGGACCCCATT
>dbeec4e8157f2a3R
ACTCACCTTAAAGGGGACCCCATTGGG
>5be7f0adcc3950dR
CAGGTAGACCTGATTGCGCAAGTCACT
>d72b73317e5f010R
CTCCAGGTAGACCTGATTGCGCAAGTC
>dfa88a344460dacR
ATCCTCCAGGTAGACCTGATTGCGCAA
>daa6576133974e6R
TCCATCCTCCAGGTAGACCTGATTGCG
>8e8cf9f6c928fafR
CAGTCCATCCTCCAGGTAGACCTGATT
>8c0be20dcdbf8d5R
GTCCAGTCCATCCTCCAGGTAGACCTG
>300d94b5702c35aR
GGGGTCCAGTCCATCCTCCAGGTAGAC
>ecb5347fdbb43e6R
GAAGGGGTCCAGTCCATCCTCCAGGTA
>3c13ce379c72a98R
TGGGAAGGGGTCCAGTCCATCCTCCAG
>2b5fb4ddacc62b3R
GCCTGGGAAGGGGTCCAGTCCATCCTC
>caf4cb0294f1d4bR
CTCGCCTGGGAAGGGGTCCAGTCCATC
>698227bd077ff9eR
GCCCTCGCCTGGGAAGGGGTCCAGTCC
>2c5763e0776993aR
ACGGCCCTCGCCTGGGAAGGGGTCCAG
>1751ddbd2782370R
CACACGGCCCTCGCCTGGGAAGGGGTC
>6097017c4283cd9R
CACCACACGGCCCTCGCCTGGGAAGGG
>5cc906a9882ae62R
GCCCACCACACGGCCCTCGCCTGGGAA
>488d3ccfc74ac0eR
CCTGCCCACCACACGGCCCTCGCCTGG
>8849b5d6062cce2R
CTGCCTGCCCACCACACGGCCCTCGCC
>95dd317c01d3169R
CTCCACCCTGTCCAAGGCCTTGTGCAT
>f59bb1ecf71ae67R
CTCCTCCACCCTGTCCAAGGCCTTGTG
>5ee463e332ba791R
GTGCTCCTCCACCCTGTCCAAGGCCTT
>8f969839cd10874R
TGGGTGCTCCTCCACCCTGTCCAAGGC
>9674371aaf3a731R
CAGAAATTTCTCAGCAGTCATCCTGGA
>a74db2a338189c8R
AAATTTCTCAGCAGTCATCCTGGAGCC
>fae540a51ecec5cR
TTTCTCAGCAGTCATCCTGGAGCCTGG
>47863b1633bbee3R
CTCAGCAGTCATCCTGGAGCCTGGGTG
>482b6bd56daeee1R
AGCAGTCATCCTGGAGCCTGGGTGCTC
>6a43bcc4a8deab4R
AGTCATCCTGGAGCCTGGGTGCTCCTC
>7eef62c76109569R
CATCCTGGAGCCTGGGTGCTCCTCCAC
>a5b2c3429627f5cR
CCTGGAGCCTGGGTGCTCCTCCACCCT
>18d262958abeb79R
GGAGCCTGGGTGCTCCTCCACCCTGTC
>81c166dc0ea9b25R
GTTCAGAAATTTCTCAGCAGTCATCCT
>f897dc312ee6c59R
CCTGTTCAGAAATTTCTCAGCAGTCAT
>20fc6c7621141f0R
GAGCCTGTTCAGAAATTTCTCAGCAGT
>1c0befdff5a4f44R
GGGGAGCCTGTTCAGAAATTTCTCAGC
>7e99b2f32c3d82aR
CTTGGGGAGCCTGTTCAGAAATTTCTC
>6e4bbbb5a457197R
AAACTTGGGGAGCCTGTTCAGAAATTT
>2b1da35b7252fbeR
CACAAACTTGGGGAGCCTGTTCAGAAA
>e35808af465ccb1R
GATCACAAACTTGGGGAGCCTGTTCAG
>ee482b213fb2f37R
CCGGATCACAAACTTGGGGAGCCTGTT
>ec8c34a7f77cd93R
TTGCCGGATCACAAACTTGGGGAGCCT
>9de7a00b2aeb51cR
GCCTTGCCGGATCACAAACTTGGGGAG
>5aa9fe35f89c1f0R
CTCGCCTTGCCGGATCACAAACTTGGG
>1e9706fc49de878R
CACCTCGCCTTGCCGGATCACAAACTT
>ba4c0e9c9dccffbR
AATCACCTCGCCTTGCCGGATCACAAA
>4f19469f1df9c7fR
GTCAATCACCTCGCCTTGCCGGATCAC
>3bf701b62c2f549R
GATGTCAATCACCTCGCCTTGCCGGAT
>97c2b94241b333cR
CCGGATGTCAATCACCTCGCCTTGCCG
>ab3a57a0e8e5388R
GCCCCGGATGTCAATCACCTCGCCTTG
>d741de755a7591dR
GGGGCCCCGGATGTCAATCACCTCGCC
>55d8fa413d9650bR
GATGGGGCCCCGGATGTCAATCACCTC
>47445be3c65df24R
CCTGATGGGGCCCCGGATGTCAATCAC
>f2afb1798adf281R
GTCCCTGATGGGGCCCCGGATGTCAAT
>bf281195ba96073R
GGTGTCCCTGATGGGGCCCCGGATGTC
>92fe2dd271bcd5cR
CAAGGTGTCCCTGATGGGGCCCCGGAT
>859ec69c0ebbcf9R
CTGCAAGGTGTCCCTGATGGGGCCCCG
>d56a61f938ced38R
GATCCGGGCAGGCAATGGGCAGCAGTT
>3fae6b10ca6069dR
CCGGGCAGGCAATGGGCAGCAGTTCTG
>5745d01d9ddeb19R
GGCAGGCAATGGGCAGCAGTTCTGCAA
>a2439aaa7850df1R
AGGCAATGGGCAGCAGTTCTGCAAGGT
>91c9686d7c84649R
CAATGGGCAGCAGTTCTGCAAGGTGTC
>674a01558e5211dR
TGGGCAGCAGTTCTGCAAGGTGTCCCT
>d8e4356d11467d6R
GCAGCAGTTCTGCAAGGTGTCCCTGAT
>b0b42de85b22df4R
GCAGTTCTGCAAGGTGTCCCTGATGGG
>fca43a5419c1729R
CTGGATCCGGGCAGGCAATGGGCAGCA
>e409dc35e2b5072R
CTCCTGGATCCGGGCAGGCAATGGGCA
>e231cfcf0636d9aR
AATCTCCTGGATCCGGGCAGGCAATGG
>9f42f9086b8d8b4R
CACAATCTCCTGGATCCGGGCAGGCAA
>dd56ec48d085a47R
CACCACAATCTCCTGGATCCGGGCAGG
>53ca981af866dc3R
CTCCACCACAATCTCCTGGATCCGGGC
>138fa8e791b615aR
CGTCTCCACCACAATCTCCTGGATCCG
>246b2d6a77d68e2R
GGGCGTCTCCACCACAATCTCCTGGAT
>ed12c3ddde0cfc8R
GGTGGGCGTCTCCACCACAATCTCCTG
>7191f20397dd6c1R
CAAGGTGGGCGTCTCCACCACAATCTC
>055345a3e9e2fbbR
GGCCAAGGTGGGCGTCTCCACCACAAT
>7a5b5a21650fe80R
AGCGGCCAAGGTGGGCGTCTCCACCAC
>265807b87abf393R
CTCAGCGGCCAAGGTGGGCGTCTCCAC
>d3bf8cbff766b57R
TCGCTCAGCGGCCAAGGTGGGCGTCTC
>5d19458fd9f2c45R
CTCTCGCTCAGCGGCCAAGGTGGGCGT
>c3781a7d76cdaa3R
GTTGGGTGACTCCTGGCTCCTCTCTCG
>9737f7a0eca2ce3R
GGGTGACTCCTGGCTCCTCTCTCGCTC
>eb968544fb1b9faR
TGACTCCTGGCTCCTCTCTCGCTCAGC
>b9e7cc40ef5eb1bR
CTCCTGGCTCCTCTCTCGCTCAGCGGC
>36f1ccf0236dd7eR
CTGGCTCCTCTCTCGCTCAGCGGCCAA
>3286632c566d638R
GCTCCTCTCTCGCTCAGCGGCCAAGGT
>33ca3cb881cbd10R
GCGCAGCATGGAGAGCGGGGGTGCCGG
>e9b09f6f22805c1R
GATGCGCAGCATGGAGAGCGGGGGTGC
>1725526b137da01R
CTTGATGCGCAGCATGGAGAGCGGGGG
>288a5c6b28f6209R
AGACTTGATGCGCAGCATGGAGAGCGG
>3b8a1fbbf7d46fcR
CTCAGACTTGATGCGCAGCATGGAGAG
>c1e389224ad5b47R
ATTCTCAGACTTGATGCGCAGCATGGA
>64142dba4005680R
CCCATTCTCAGACTTGATGCGCAGCAT
>e71675f42d69c11R
TTCCCCATTCTCAGACTTGATGCGCAG
>36e69d845b61744R
CTGTTCCCCATTCTCAGACTTGATGCG
>5b93e1c09c980a0R
GGCCTGTTCCCCATTCTCAGACTTGAT
>71e8a59cf61a744R
GAAGGCCTGTTCCCCATTCTCAGACTT
>b3878c210a77665R
TAGGAAGGCCTGTTCCCCATTCTCAGA
>f78687c109850c0R
CAGTAGGAAGGCCTGTTCCCCATTCTC
>4f67ad8e4b58e5eR
CATCAGTAGGAAGGCCTGTTCCCCATT
>85caf93a061ea26R
CATCATCAGTAGGAAGGCCTGTTCCCC
>3860f7e1a2bef57R
CTGCATCATCAGTAGGAAGGCCTGTTC
>e0aeaa379938e26R
AGGCTGCATCATCAGTAGGAAGGCCTG
>b0862365ac48319R
GTCAGGCTGCATCATCAGTAGGAAGGC
>18084693b55b36cR
GTTGTCAGGCTGCATCATCAGTAGGAA
>f015bdbc452838eR
GGTGTTGTCAGGCTGCATCATCAGTAG
>dbdc03e3223fe7aR
AATGGTGTTGTCAGGCTGCATCATCAG
>5accde2a95621b3R
CCCAATGGTGTTGTCAGGCTGCATCAT
>5c1ae19eff4c808R
GTCCCCAATGGTGTTGTCAGGCTGCAT
>018c8d43193c4f7R
CACGTCCCCAATGGTGTTGTCAGGCTG
>127966b61bfc92bR
TCGCACGTCCCCAATGGTGTTGTCAGG
>1e5d371aec4f31bR
AGCTCGCACGTCCCCAATGGTGTTGTC
>95d8cbf2f215a03R
CAGAGCTCGCACGTCCCCAATGGTGTT
>f8e9189d1906aa7R
TAGCAGAGCTCGCACGTCCCCAATGGT
>acfe7c0e12cf9feR
CGCTAGCAGAGCTCGCACGTCCCCAAT
>d49807d152485faR
CTGCGCTAGCAGAGCTCGCACGTCCCC
>be9dd9c3d456ea6R
GGCCTGCGCTAGCAGAGCTCGCACGTC
>6e66472c16aeedfR
GATCTCAAAGGCAGAGGCATCCATGAC
>4265a0d83fe6946R
CTCAAAGGCAGAGGCATCCATGACCCT
>3c85183821797e9R
AAAGGCAGAGGCATCCATGACCCTGGC
>f9d5ad32ba7080cR
GGCAGAGGCATCCATGACCCTGGCCTG
>88f48a7b231f4deR
AGAGGCATCCATGACCCTGGCCTGCGC
>5d9fbc5f183f731R
GGCATCCATGACCCTGGCCTGCGCTAG
>c6d7f72d7d1b73eR
ATCCATGACCCTGGCCTGCGCTAGCAG
>e9863c1202e515bR
CATGACCCTGGCCTGCGCTAGCAGAGC
>9625baec5efb17bR
GACCCTGGCCTGCGCTAGCAGAGCTCG
>6436bc37486a160R
GAAGATCTCAAAGGCAGAGGCATCCAT
>2317811a07b4035R
GCTGAAGATCTCAAAGGCAGAGGCATC
>41c2bbb77df2f93R
TGTGCTGAAGATCTCAAAGGCAGAGGC
>c6037f2c0d608eaR
GAATGTGCTGAAGATCTCAAAGGCAGA
>cd6d5b7cc2c46b8R
CGGGAATGTGCTGAAGATCTCAAAGGC
>b5c0e9e1cbd6798R
GGGCGGGAATGTGCTGAAGATCTCAAA
>dc4ce5b1aceb3acR
GGTGGGCGGGAATGTGCTGAAGATCTC
>4ec8cfa932cbd81R
GAGGGTGGGCGGGAATGTGCTGAAGAT
>4ea61691feb614aR
GTAGAGGGTGGGCGGGAATGTGCTGAA
>b2ec2571d717530R
CTGGTAGAGGGTGGGCGGGAATGTGCT
>523cf6ee47dc1ccR
GTCCTGGTAGAGGGTGGGCGGGAATGT
>f92c2d8dd0e9f80R
ATCGTCCTGGTAGAGGGTGGGCGGGAA
>907cb89b9fdac48R
TGTATCGTCCTGGTAGAGGGTGGGCGG
>31ddf4f925bf4e3R
GAGTGTATCGTCCTGGTAGAGGGTGGG
>999a82f52e6eb7eR
CGTGAGTGTATCGTCCTGGTAGAGGGT
>87131e5f79bc1d6R
CAGCGTGAGTGTATCGTCCTGGTAGAG
>2eb6decc6e41498R
CTGCAGCGTGAGTGTATCGTCCTGGTA
>fbd9c4d70d22f00R
AGCCTGCAGCGTGAGTGTATCGTCCTG
>2f82ca4a53d499cR
TGCAGCCTGCAGCGTGAGTGTATCGTC
>0fec6d892749e35R
GCCTGCAGCCTGCAGCGTGAGTGTATC
>58d89aea87012d7R
AAGGCCTGCAGCCTGCAGCGTGAGTGT
>daa6d91ab561978R
CACAAGGCCTGCAGCCTGCAGCGTGAG
>2694cf2f2047a4aR
GGGCACAAGGCCTGCAGCCTGCAGCGT
>9fdd2e02b91c983R
TTTGGGCACAAGGCCTGCAGCCTGCAG
>83ab150cb63e9dfR
TGCTTTGGGCACAAGGCCTGCAGCCTG
>96b040fa76238b7R
TGCTGCTTTGGGCACAAGGCCTGCAGC
>27a4c0b0969768dR
CAGTGCTGCTTTGGGCACAAGGCCTGC
>8eafa9eb666f714R
CAGCAGTGCTGCTTTGGGCACAAGGCC
>dedc287c192687dR
CAGCAGCAGTGCTGCTTTGGGCACAAG
>89803617823a5cbR
CCGCAGCAGCAGTGCTGCTTTGGGCAC
>cb11cbf843f4716R
TGCCCGCAGCAGCAGTGCTGCTTTGGG
>130481a851d36b4R
GCGTGCCCGCAGCAGCAGTGCTGCTTT
>a8f44161f491b80R
TCGGCGTGCCCGCAGCAGCAGTGCTGC
>00ce26b69704df9R
GGCTCGGCGTGCCCGCAGCAGCAGTGC
>415754f24b77cd6R
CGGGGCTCGGCGTGCCCGCAGCAGCAG
>a732b0580ec1b48R
CTTCGGGGCTCGGCGTGCCCGCAGCAG
>61958690910cd12R
GGACTTCGGGGCTCGGCGTGCCCGCAG
>fa3dec1922bc2e8R
GCTGGACTTCGGGGCTCGGCGTGCCCG
>7aec34564034443R
CAGGCTGGACTTCGGGGCTCGGCGTGC
>f003965f25c9f59R
TTTCAGGCTGGACTTCGGGGCTCGGCG
>fde2b2464d03367R
GAATTTCAGGCTGGACTTCGGGGCTCG
>c5081c5891a4caeR
acTGAATTTCAGGCTGGACTTCGGGGC
>32d66950b23bbf3R
aggacTGAATTTCAGGCTGGACTTCGG
>678cdec8864f36bR
accaggacTGAATTTCAGGCTGGACTT
>4e670353fc867d7R
gggaccaggacTGAATTTCAGGCTGGA
>f8d62acc9fe3998R
acagggaccaggacTGAATTTCAGGCT
>d15f341e7c5ab7eR
gggacagggaccaggacTGAATTTCAG
>556c9854fdf9973R
accgggacagggaccaggacTGAATTT
>1926701fb0f2f56R
gggaccgggacagggaccaggacTGAA
>1531b8346988829R
accgggaccgggacagggaccaggacT
>8e12394ab6b9d9eR
gggaccgggaccgggacagggaccagg
>0f49d83e6eefce3R
Aggacagggactggggccgggaccggg
>b36a420931bf3a2R
TCCAggacagggactggggccgggacc
>71ff73fd98a003cR
GGGTCCAggacagggactggggccggg
>7888a513b0d86e9R
ACTGGGTCCAggacagggactggggcc
>ffedf964dae4eeeR
GGGACTGGGTCCAggacagggactggg
>8006117d36c97d5R
GCTGGGACTGGGTCCAggacagggact
>6d10b4b32df1c97R
GGGGCTGGGACTGGGTCCAggacaggg
>3d09ac16604cc53R
TTGGGGGCTGGGACTGGGTCCAggaca
>8f0d0565e067d94R
TTATTGGGGGCTGGGACTGGGTCCAgg
